ArgoTests/Models/EstablishmentTestModel.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  EstablishmentTestModel.swift
+//  Argo
+//
+//  Created by David on 05/11/2015.
+//  Copyright © 2015 thoughtbot. All rights reserved.
+//
+
+import Foundation
+import Argo
+import Curry
+
+struct Establishment {
+  let id: Int
```

Comment:
Name Format Violation: Variable name should be between 3 and 40 characters in length: 'id'